### PR TITLE
ProductsListing Page UI+API integration of /product API

### PR DESCRIPTION
**Pull request by ultronjs**
Added Product Listing Page
API for Product
Added Products in DB

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Good Structure
clean code

File: src/components/ProductContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import ProductCard from './ProductCard';
+import axios from 'axios'
+
+function ProductContainer() {
+    const [product,setProduct] = useState([])
+    useEffect(()=>{
+      (async function (){
+        try{
+        const {products} = await axios.get(
+          "/api/products").then(response => response.data);
```
Comment (JantuDeb) [inline src/components/ProductContainer.jsx:16]: No need to use `then(response => response.data)` 
```
const {products} = await axios.get(
          "/api/products").data
```

**Review by JantuDeb — APPROVED** [review]

File: src/components/ProductContainer.jsx (hunk 1 above)
Comment (ultronjs) [reply src/components/ProductContainer.jsx:16]: Cool Thanks

**Comment by ultronjs** [comment]
Merging after 2 approval

**Pull request merged**